command.go

```diff
@@ -341,6 +341,10 @@ func (c *Command) Find(arrs []string) (*Command, []string, error) {
 		return c.Root(), arrs, nil
 	}
 
+	for len(arrs) > 0 && len(arrs[0]) == 0 {
```

Comment:
Stripping leading empty arguments?  Is this related to the bad problem or just something else you found on the way?